<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文档修改神器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="FileTable">
     <attribute name="title">
      <string>文件夹操作</string>
     </attribute>
     <widget class="QPushButton" name="FileButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>290</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>FileButton</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Edit_Path">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Edit_Source">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Edit_Target">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件夹所在路径</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>源数据</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>目标数据</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_FilePath">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>请选择文件所在路径下任意一个文件以便我们能顺利找到它们</string>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>文件后缀</string>
      </property>
     </widget>
     <widget class="QComboBox" name="ComboBox_Suffix">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>72</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string notr="true">*.doc</string>
      </property>
      <item>
       <property name="text">
        <string>*.doc</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>*.docx</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>*.rtf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>*.xls</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>*.xlsx</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>*.txt</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="ContentTable">
     <attribute name="title">
      <string>文件内容操作</string>
     </attribute>
     <widget class="QPushButton" name="ContentButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>290</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ContentButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>路径</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
